docs/index.rst

```diff
@@ -36,10 +36,9 @@ pages for each source:
 * :doc:`data_sources/sec10k`
 * :doc:`data_sources/vcerare`
 
-PUDL's clean and complete versions of these data sources are stored in the
-``pudl.sqlite`` database. Larger datasets, like EPA CEMS, are stored in parquet files.
-To get started using PUDL data, visit our :doc:`data_access` page, or continue reading
-to learn more about the PUDL data processing pipeline.
+PUDL's processed versions of these data sources are distributed primarily as Parquet
+files. To get started using PUDL data, visit our :doc:`data_access` page, or continue
+reading to learn more about the PUDL data processing pipeline.
```

Comment:
Reduced emphasis on SQLite since Parquet is the only place for a growing proportion of our data, and more and more the canonical source.

Reply:
Good call.